jenkins/client.sls

```diff
@@ -0,0 +1,40 @@
+{% from "jenkins/map.jinja" import client with context %}
+{%- if client.enabled %}
+
+jenkins_client_install:
+  pkg.installed:
+  - names: {{ client.pkgs }}
+
+jenkins_client_dirs:
+  file.directory:
+  - name: /srv/salt/env/dev/_jenkins_jobs
```

Comment:
This is sooo ugly location, also jenkins.client doesn't need to run on salt master to use this path root dedicated to setup salt master env.
Maybe use `/var/tmp/salt_jenkins_jobs` or don't even use that temp file.